test/bigint.cc

```diff
@@ -1,4 +1,3 @@
-#define NAPI_EXPERIMENTAL
```

Comment:
I think this will cause the tests not to run due to this later check #if (NAPI_VERSION > 2147483646)

Reply:
🤔 maybe an option to control whether `NAPI_EXPERIMENTAL` should be defined or not might be needed.

Reply:
If I remember correctly we had it enabled by default so that we'd run the tests in the most common use case (running on master) and then then CI sets the env variable so that the tests don't run when they run on earlier node version (ex Node.js v8)

Reply:
But if `NAPI_EXPERIMENTAL` is defined the binding.node would mark napi_bigint related symbol as required, which should definitely fail on Node.js v8.

Reply:
I noticed `npm_config_NAPI_VERSION` in `test/index.js` which would disable bigint tests. AFAIKT, it doesn't affect compilation right? Then requiring binding.node would fail on Node versions that don't have bigint NAPIs.

Reply:
https://travis-ci.com/legendecas/node-addon-api/builds/125931273

Used node major version to determine if experimental tests should be ran.

In Node.js v8 bigint tests were ignored.
In Node.js v10 bigint tests were ran.